packages/extension-ui/src/Popup/Authorize/index.tsx

```diff
@@ -14,9 +14,9 @@ export default function Authorize (): React.ReactElement {
   const requests = useContext(AuthorizeReqContext);
 
   return (
-    <>
-      <Scroll isLastRequest={requests.length === 1}>
-        <Header text={t<string>('Authorize')} />
+    <Scroll isLastRequest={requests.length === 1}>
```

Comment:
Just worried the removal of the fragment here breaks stuff - there are a couple `> div` which was hugely problematic styling-wise in the past.